typos.toml

```diff
@@ -0,0 +1,10 @@
+# See https://github.com/crate-ci/typos/blob/master/docs/reference.md to configure typos
+[files]
+extend-exclude = [
+    "*.svg",
+    "*.drawio",
+]
+[default.extend-words]
+BA = "BA"
+Ded = "Ded"
+EDE = "EDE"
```

Comment:
Please add an empty line at the end of the file.

Reply:
done, but where is this guideline from?

Reply:
It seems that there is no explicit guideline, it is a default convention of the Karmada repository.

Reply:
> it is a default convention of the Karmada repository.

No, it's not just `a default convention of the Karmada repository`.

The reason why these lines are there is because without them, there would be a lot of errors misreported.

Reply:
the reason for filter `EDE = "EDE"` is:

![image](https://github.com/karmada-io/website/assets/30589999/2bc73921-9c1d-45e4-9086-cc3de0405000)

the reason for filter `*.drawio` is:

![image](https://github.com/karmada-io/website/assets/30589999/5c1136bb-ebcd-4dd5-9445-52ae763e1958)

they are all misreported, should be filtered.

Reply:
I guess you're replying to my other question, right?